static/js/setup.js

```diff
@@ -67,15 +67,6 @@ $(() => {
             // process of executing a password change.
             return;
         }
```

Comment:
This `return` block above needs to come along as well if we're doing this refactor -- the whole point of that `return` block is to avoid the below moved logic firing incorrectly.  (I think we also get to delete the whole `ajaxError` handler here once we do so, though of course more code ends up needing to move for the `password_change_in_progress` checks).

Also @andersk can you help me confirm that

Reply:
https://github.com/zulip/zulip/commit/fa2118ce90246c164fbd204a57c3118325b9cb12 is the commit.